web/client/plugins/drawer/Menu.jsx

```diff
@@ -58,11 +58,18 @@ var Menu = React.createClass({
         );
     },
     renderButtons() {
-        return this.props.children.map((child) => (
-            <Button key={child.props.eventKey} bsSize="large" className={(child.props.buttonConfig && child.props.buttonConfig.buttonClassName) ? child.props.buttonConfig.buttonClassName : "square-button"} onClick={this.props.onChoose.bind(null, child.props.eventKey)} bsStyle={this.props.activeKey === child.props.eventKey ? 'default' : 'primary'}>
-                {child.props.glyph ? <Glyphicon glyph={child.props.glyph} /> : child.props.icon}
-            </Button>
-        ));
+        return this.props.children.map((child) => {
+            const button = (<Button key={child.props.eventKey} bsSize="large" className={(child.props.buttonConfig && child.props.buttonConfig.buttonClassName) ? child.props.buttonConfig.buttonClassName : "square-button"} onClick={this.props.onChoose.bind(null, child.props.eventKey)} bsStyle={this.props.activeKey === child.props.eventKey ? 'default' : 'primary'}>
+                        {child.props.glyph ? <Glyphicon glyph={child.props.glyph} /> : child.props.icon}
+                    </Button>);
+            const tooltip = <Tooltip key={"tooltip." + child.props.eventKey} id={"tooltip." + child.props.eventKey}>{child.props.buttonConfig.tooltip || ''}</Tooltip>;
+            return child.props.buttonConfig.tooltip ? (
+                <OverlayTrigger placement={"top"} key={"overlay-trigger." + child.props.eventKey}
```

Comment:
const tooltip = <Tooltip key={"tooltip." + child.props.eventKey} id={"tooltip." + child.props.eventKey}><Message msgId={child.props.buttonConfig.tooltip}/></Tooltip>
child.props.buttonConfig.tooltip is correct but the tooltip is empty... 
renderButtons() {
        return this.props.children.map((child) => {
            const button = (<Button key={child.props.eventKey} bsSize="large" className={(child.props.buttonConfig && child.props.buttonConfig.buttonClassName) ? child.props.buttonConfig.buttonClassName : "square-button"} onClick={this.props.onChoose.bind(null, child.props.eventKey)} bsStyle={this.props.activeKey === child.props.eventKey ? 'default' : 'primary'}>
                        {child.props.glyph ? <Glyphicon glyph={child.props.glyph} /> : child.props.icon}
                    </Button>);
            const tooltip = <Tooltip key={"tooltip." + child.props.eventKey} id={"tooltip." + child.props.eventKey}><Message msgId={child.props.buttonConfig.tooltip}/></Tooltip>;
            return child.props.buttonConfig.tooltip ? (
                <OverlayTrigger placement={"bottom"} key={"overlay-trigger." + child.props.eventKey}
                    overlay={tooltip}>
                    {button}
                </OverlayTrigger>
            ) : button;
        });
    },